### DISCUSSION TITLE
2022/06/29/%E3%80%8EGitHub%E3%80%8F%E5%AD%A6%E7%94%9F%E8%BA%AB%E4%BB%BD%E8%AE%A4%E8%AF%81%E9%97%AE%E9%A2%98/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#18] [General]
# 2022/06/29/%E3%80%8EGitHub%E3%80%8F%E5%AD%A6%E7%94%9F%E8%BA%AB%E4%BB%BD%E8%AE%A4%E8%AF%81%E9%97%AE%E9%A2%98/

因为 Copilot 准备收费了，而学生还是可以免费用的，所以去申请了好几次教育认证，今天终于把 GitHub 教育认证弄好了（而且是秒过） 下面是鄙人的个人经历，可供参考，但是我也不一定包你能过，我只能说我是这样过的  下面开始，并且全程不要用梯子  登录 首先前往 https:&#x2F;&#x2F;educati

https://nx-official.github.io/2022/06/29/%E3%80%8EGitHub%E3%80%8F%E5%AD%A6%E7%94%9F%E8%BA%AB%E4%BB%BD%E8%AE%A4%E8%AF%81%E9%97%AE%E9%A2%98/

**Comment by ZhangStudyLife** [comment]
妙啊，以前我还想GitHub的copilot这么贵难道您直接买啦？

**Comment by ZhangStudyLife** [comment]
我失败了，全程没开东西，但是没上传学信网的，而是录取通知书的英文翻译经过手动修图以后的图片，可惜了，打算再试试

**Reply by ZhangStudyLife** [reply]
但是我也怀疑是不是自己github设置有问题，什么启用双因素身份验证

**Reply by nickxudotme** [reply]
现在要求看上去很严了，只能拍照，而且还有定位问题什么一大堆

我最近认证过期了，想续，但是他说我不在学校，我给的理由是在放假，但是那个 bot 死活看不懂，算了，等返校再搞了

所以我现在的确是用的付费的 Copilot 哈哈哈，每月 10 刀这两个月还勉勉强强能接受

就我身边的人来说，也有拍学生证认证成功的，反正是所有能「证明」你的身份的都试一下吧

至于你，我觉得你可以等下个月入学了再试试

**Reply by ZhangStudyLife** [reply]
谢谢，手机也尝试各种拍照，现在只能放弃了，大抵是我现在还不是纯正的大学生吧哈哈哈
